QemuPkg/FrontPageButtons/FrontPageButtons.inf

```diff
@@ -0,0 +1,52 @@
+## @file FrontpageButtons.inf
+#
+# This module installs the MsButtonServicesProtocol and reports the requested state of Vol+ and
+# Vol- as indicated by the smbios table type3 record version string.
+#
+# If the Type3.version string is:
+#     "Vol+"  -- Return Vol+ pressed.
+#     "Vol-"  -- Return Vol- pressed.
+#     any other string, or no string, return neither pressed
+#
+# Copyright (C) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+##
+
+[Defines]
+  INF_VERSION                    = 0x00010005

```

Comment:
nit: for new inf, should we use a newer `INF_VERSION`?

Reply:
Fixed